lib/themes/dosomething/paraneue_dosomething/templates/system/html.tpl.php

```diff
@@ -39,6 +39,26 @@
 </head>
 
 <body class="<?php print $classes; if ($variables['is_affiliate']) print ' -affiliate'; ?>" <?php print $attributes;?>>
+  <?php if ($variables['enable_facebook_login']): ?>
+    <script>
+      window.fbAsyncInit = function() {
+        FB.init({
+          appId      : '105775762330',
```

Comment:
Should this be a configuration variable so it can point to different apps per environment?